src/App/Services/MobilePlatformUtilsService.cs

```diff
@@ -269,6 +270,12 @@ public async Task<bool> IsBiometricIntegrityValidAsync(string bioIntegritySrcKey
                 {
                     fallback?.Invoke();
                 }
+                if (result.Status == FingerprintAuthenticationResultStatus.TooManyAttempts
+                    && logOutOnTooManyAttempts)
+                {
+                    await ShowDialogAsync(AppResources.AccountLoggedOutBiometricExceeded, AppResources.TooManyAttempts, AppResources.Ok);
+                    _messagingService.Send(AccountsManagerMessageCommands.LOGOUT, new Tuple<string, bool, bool>(null, true, false));
```

Comment:
⛏️ You can skip the `Tuple` parameter given that those are the default values when the extras parameter is `null` as you can see [here](https://github.com/bitwarden/mobile/blob/master/src/App/Utilities/AccountManagement/AccountsManager.cs#L153).